docs/source/process.rst

```diff
@@ -115,33 +83,38 @@ By default, echopype uses the following for calibration:
 
 - EK60: Environmental parameters saved with the data files
 
-- AZFP: salinity = 29.6 PSU, pressure = 60 dbar,
+- EK80: Environmental parameters saved with the data files
+
+- AZFP: Salinity and pressure provided by the user,
   and temperature recorded at the instrument
 
 These parameters should be overwritten when they differ from the actual
 environmental condition during data collection.
-To update these parameters, simply do the following *before*
-calling ``ed.calibrate()``:
+To update these parameters, simply pass in the environmental parameters
+as a dictionary while calling ``ep.calibrate.compute_Sv()`` like so:
```

Comment:
These parameters should be overwritten --> These parameters **can be** overwritten